app/util/api/confluence_clients.py

```diff
@@ -124,6 +124,7 @@ def search(self, cql, cqlcontext=None, expand=None, start=0, limit=500):
 
         return search_results_list
 
+    @retry()
```

Comment:
Pls double check the behavior when actually there is Remote API is disabled.
With retry, we still should report a correct error, but after 4 retries.

Reply:
Checked - an error appears:
`Exception: Confluence Remote API (XML-RPC & SOAP) is disabled. For further processing enable Remote API via General Configuration - Further Configuration - Remote API. Response code:[403]`